dev/io.openliberty.org.jboss.resteasy.common.ee11/src/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -69,20 +70,17 @@ public class ContextParameterInjector implements ValueInjector {
     private volatile boolean outputStreamWasWritten = false;
 
     static {
-        constructor = AccessController.doPrivileged(new PrivilegedAction<Constructor<?>>() {
-            @Override
-            public Constructor<?> run() {
-                try {
-                    Class.forName("jakarta.servlet.http.HttpServletResponse", false,
-                            Thread.currentThread().getContextClassLoader());
-                    Class<?> clazz = Class.forName("org.jboss.resteasy.core.ContextServletOutputStream");
-                    return clazz.getDeclaredConstructor(ContextParameterInjector.class, OutputStream.class);
-                } catch (Exception e) {
-                    return null;
-                }
-            }
-        });
-        // liberty change start
+        Constructor<?> c;
+        try {
+            Class.forName("jakarta.servlet.http.HttpServletResponse", false,
+                    Thread.currentThread().getContextClassLoader());
+            Class<?> clazz = Class.forName("org.jboss.resteasy.core.ContextServletOutputStream");
+            c = clazz.getDeclaredConstructor(ContextParameterInjector.class, OutputStream.class);
+        } catch (Exception ignore) {
+            c = null;
+        }
+        constructor = c;
+		// liberty change start
         myClassLoader = AccessController.doPrivileged(new PrivilegedAction<ClassLoader>() {
```

Comment:
```suggestion
        myClassLoader = ContextParameterInjector.class.getClassLoader();
```
is what this becomes after removing the doPriv.